client/src/components/Form/Elements/FormText.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+    <div>
+        <b-alert class="mt-2" v-if="errorMessage" :show="dismissCountDown" variant="info" @dismissed="resetAlert">
+            {{ errorMessage }}
+        </b-alert>
+        <b-row align-v="center">
+            <b-col>
+                <!-- type        <str> "text" or "password" -->
```

Comment:
I suggest to move these comments to the props, since we do it in other files like that too for now.